contracts/Tokenomics.sol

```diff
@@ -283,8 +283,8 @@ contract Tokenomics is TokenomicsConstants {
     event EpochSettled(uint256 indexed epochCounter, uint256 treasuryRewards, uint256 accountRewards,
         uint256 accountTopUps, uint256 effectiveBond, uint256 returnedStakingIncentive, uint256 totalStakingIncentive);
     event TokenomicsImplementationUpdated(address indexed implementation);
-    event InflationPerSecondFractionsUpdated(uint256 inflationPerSecond, uint256 maxBond, uint256 maxBondFraction,
-        uint256 topUpComponentFraction, uint256 topUpAgentFraction, uint256 stakingFraction);
+    event InflationPerSecondFractionsUpdated(uint256 inflationPerSecond, uint256 effectiveBond, uint256 returnedStakingIncentive,
```

Comment:
what happened to maxBond?

Reply:
It's the same as the updated `effectiveBond` and will be emitted during the checkpoint epoch settlement.